src/components/MarkDown/__stories__/index.stories.mdx

```diff
@@ -10,26 +16,29 @@ import { simple, withMultipleParagraphs } from './markdown_example'
   A wrapper for [react-markdown](https://github.com/remarkjs/react-markdown)
 </Description>
 
-## Basic Usage
-
-<Canvas isColumn>
-  <Story name="basic">
+<Canvas>
+  <Story name="Basic">
     <MarkDown source={simple} linkTarget="_blank" />
   </Story>
 </Canvas>
 
 ## Inline
 
 <Description>
-  You can make the paragraphs inside the source render inline by settting the
+  You can make the paragraphs inside the source render inline by setting the
   `inline prop`
 </Description>
 
-<Canvas isColumn>
-  <Story name="with-paragraphs">
+<Canvas>
+  <Story name="With paragraphs">
     <MarkDown source={withMultipleParagraphs} />
   </Story>
-  <Story name="with-paragraphs-inline">
+</Canvas>
+
+<Canvas>
+  <Story name="With paragraphs inline">
     <MarkDown source={withMultipleParagraphs} inline />
   </Story>
 </Canvas>
+
```

Comment:
Both properties `linkTarget` and `escapeHtml ` are missing in examples and no comments on them so we don't know what they can be used for